version.sh

```diff
@@ -1 +1 @@
-export ARTIFACT_GO_VERSION='1.12.7'
+export ARTIFACT_GO_VERSION='1.14.10'
```

Comment:
if using 1.14.x in travis.yaml, we should laso update this
```suggestion
export ARTIFACT_GO_VERSION='1.14.x'
```